docs/docs/guides/develop/automate/continuous-integration.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Continuous Integration (CI)
+titleTemplate: ":title | Develop | Tuist"
+description: Learn how to use Tuist in your CI workflows.
+---
+
+# Continuous Integration (CI)
+
+You can use Tuist in [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) environments. The following sections provide examples of how to do this on different CI platforms.
+
+## Examples
+
+To run Tuist commands in your CI workflows, you’ll need to install it in your CI environment. The following sections provide examples of how to do this on different CI platforms.
```

Comment:
```suggestion
To run Tuist commands in your CI workflows, you’ll need to install it in your CI environment.
```

The "following sections..." appears twice here (it's in the `CI` header as well)